packages/rocketchat-integrations/lib/rocketchat.js

```diff
@@ -0,0 +1,67 @@
+RocketChat.integrations = {
```

Comment:
Don't we usually capitalize first letter on those? Eg. `RocketChat.Cache`, `RocketChat.SlackBridge`, `RocketChat.Settings`

Although there's also an example for `RocketChat.models`. Just curious